packages/core-js/modules/web.structured-clone.js

```diff
@@ -0,0 +1,11 @@
+var WeakMap = require('../modules/es.weak-map');
```

Comment:
Instead of direct dependencies, for preventing bloating of bundles, features like `WeakMap` should be placed to entry points (like `stable/structured-clone` and in the code use `internal/get-built-in` helper for getting `WeakMap` constructor and tools like `preset-env` will be able to avoid `es.weak-map` in engines where it's supported.

Reply:
See, for example, how `Map` used in `esnext.array.unique-by`.